src/expr/relation/mod.rs

```diff
@@ -94,6 +94,8 @@ pub enum RelationExpr {
         /// The input columns are often discarded and can be very
         /// expensive to reproduce, so restricting what we produce
         /// as output can be a substantial win.
+        ///
+        /// See expr::transform::Demand for more details.
```

Comment:
If you format like so

```
[`expr::transform::Demand`]
```

it'll get automatically linked at https://mtrlz.dev/api/rust/expr/enum.RelationExpr.html.